openzwavemqtt/models/node.py

```diff
@@ -220,3 +226,31 @@ def create_collections(self):
             "association": ItemCollection(OZWNodeAssocation),
             "statistics": OZWNodeStatistics,
         }
+
+    def get_command_class(
+        self, command_class_id: CommandClass, instance_id=None
+    ) -> bool:
+        """Return a specific CommandClass on this node (if exists)."""
+        # pylint: disable=no-member
+        for instance in self.instances():
+            if not (instance_id is None or instance.instance == instance_id):
```

Comment:
It's easier to read if we remove the parenthesis and adjust the expression accordingly.